donormotor/__init__.py

```diff
@@ -101,7 +101,7 @@ def wants_json(self):
 app.static_folder = 'static'
 
 if app.config['PROXY_FIX']:
-    from werkzeug.contrib.fixers import ProxyFix
+    from werkzeug.middleware.proxy_fix import ProxyFix
     app.wsgi_app = ProxyFix(app.wsgi_app,
                             num_proxies=app.config['PROXY_FIX_NUM_PROXIES'])
```

Comment:
The syntax for this will need to change. See https://github.com/wuvt/trackman/commit/c6b6ceda7deab5232bb1dbc0d6e7eb1279641d07